test/completion.cjs

```diff
@@ -594,6 +594,22 @@ describe('Completion', () => {
 
           r.logs.should.have.length(0);
         });
+
+        it('works if strip-dashed is set to `true`', () => {
+          const r = checkUsage(
+            () =>
+              yargs([...firstArguments, './completion', '--foo', '--'])
+                .parserConfiguration({'strip-dashed': true})
```

Comment:
Notice the `strip-dashed': true` here. We are just testing to ensure this option does _not_ do anything to our `completion` command.